### PR TITLE
Css upgrade & description placeholder

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -13,11 +13,24 @@
   justify-content: space-evenly;
 }
 
+main {
+  display: grid;
+  grid-template-areas:
+    'canvas desc'
+    'canvas editor';
+  gap: 1rem;
```
Comment (songofroland) [inline src/App.css:21]: this does not need to be relative I think

File: src/App.css [hunk 2]
```diff
@@ -79,7 +93,13 @@ textarea {
   font-size: 1.3em;
 }
 
-@media (max-width: 480px) {
+@media (max-width: 640px) {
+  main {
+    grid-template-areas:
+      'desc'
+      'canvas'
```
Comment (songofroland) [inline src/App.css:100]: same

File: src/App.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ import createFrames from './logic/graphFrames';
 import mapGraph from './logic/graphMapping';
 import labelGraph from './logic/graphLabeling';
 import { DijkstraTracer } from './logic/dijkstra';
+import Desc from './components/Desc';
```
Comment (songofroland) [inline src/App.tsx:15]: `Desc` -> `Description`
otherwise it makes me think of descending

**Review by songofroland — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (wilmhit) [reply src/App.css:21]: r stands for root

Comment (wilmhit) [reply src/App.css:21]: its better if it scales with the device, not font size in .app

File: src/App.css [hunk 4]
```diff
@@ -13,11 +13,24 @@
   justify-content: space-evenly;
 }
 
+main {
+  display: grid;
+  grid-template-areas:
```
[the remark this answers is not in the document]
Comment (wilmhit) [reply src/App.css:18]: it's not the same thing. Using areas can reorder elements. `grid-template-columns` and rows can also do this, but I would need to do `grid-column-start`, `grid-column-end`, `grid-row-start`, `grid-row-end` on each child twice (one normal and one mobile).

File: src/App.css (hunk 1 above)
Comment (wilmhit) [reply src/App.css:21]: ill remove it and fix it with `space-evenly` instead

**Review by songofroland — APPROVED** [review]

**Pull request merged**